src/client/index.ts

```diff
@@ -17,12 +17,36 @@ const socket = io(window.location.origin, {
 const FILE_BEGIN = '\u001b[5i';
 const FILE_END = '\u001b[4i';
 
+//NOTE text selection on double click or select
```

Comment:
Can you put this function in a separate file and import it

Reply:
Thanks for this great suggestion but then we are looking at passing **term** (terminal) object to that file, will that be okay?
Accessed here:
```
   if (term.hasSelection()) 
      copyToClipboard(term.getSelection())
```

Reply:
you can leave everything in the `socker.on("connect")` as is but move `copyToClipboard` to an external file and export it. and then import it at the top of this file as `import copyToClipboard from "./clipboard";`

Reply:
Okay, so you are saying only the function should be in different file? Yeah that can be done, we'll do that. Thanks.

Reply:
In the latest update

- Copy To clipboard function separated

- Yarn lint is done